src/ImageIO/CGImageDestination.cs

```diff
@@ -470,5 +495,16 @@ public bool CopyImageSource (CGImageSource image, CGImageDestinationOptions opti
 					o.Dispose ();
 			}
 		}
+
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
+		[DllImport (Constants.ImageIOLibrary)]
+		static extern void CGImageDestinationAddAuxiliaryDataInfo (IntPtr /* CGImageDestinationRef* */ idst, IntPtr /* CFStringRef* */ auxiliaryImageDataType, IntPtr /* CFDictionaryRef* */ auxiliaryDataInfoDictionary);
+
+		public void AddAuxiliaryDataInfo (CGImageDestination dest, CGImageAuxiliaryDataType auxiliaryImageDataType, CGImageAuxiliaryDataInfo auxiliaryDataInfo)
```

Comment:
Oops, missed that earlier. Copy `[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]` to the public method, otherwise the developer won't know if it's available.